frontend/public/components/factory/list.tsx

```diff
@@ -19,6 +19,7 @@ import { ingressValidHosts } from '../ingress';
 import { routeStatus } from '../routes';
 import { secretTypeFilterReducer } from '../secret';
 import { bindingType, roleType } from '../RBAC';
+// import { getRole } from '../node';
```

Comment:
@spadgett we tried to import `getRole` to sort by, but it didn't work. Should that have worked?